### PR TITLE
feat(@deip/vue-layout-schema): new package merged from schema-rendere…

### PR DESCRIPTION
**Pull request by euheny**
…r and schema builder + repo restructure

### CODE REVIEW

File: packages/common/vue-layout-schema/lib/parser/vls-parser.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ import {
 
 import { cloneDeep, merge } from '@deip/toolbox/lodash';
 
-export const SchemaRenderer = {
-  name: 'SchemaRenderer',
+export const VlsParser = {
+  name: 'VueLayoutSchemaParser',
```
Comment (elena-vlasova) [inline packages/common/vue-layout-schema/lib/parser/vls-parser.js:19]: Maybe VlsParser?

File: packages/common/vue-layout-schema/lib/builder/components/block-settings/vls-builder-block-settings.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ import { convertBlockPropsForCanvas } from '../../utils/helpers';
 
 const PERM_DISABLED = ['proxyProps'];
 
-export const SchemaBuilderBlockSettings = {
+export const VlsBuilderBlockSettings = {
   name: 'SchemaBuilderBlockSettings',
```
[the remark this answers is not in the document]
Comment (euheny) [reply packages/common/vue-layout-schema/lib/builder/components/block-settings/vls-builder-block-settings.js:25]: error

File: packages/common/vue-layout-schema/lib/builder/components/canvas/vls-builder-canvas.js [hunk 3]
```diff
@@ -9,14 +9,14 @@ import {
...
+import './vls-builder-canvas.scss';
 
 import { wrapInArray, pascalCase, deepFindParentByValue } from '@deip/toolbox';
 import draggable from 'vuedraggable';
 import { SchemeView } from '../../mixins';
 import { getters } from '../../store';
 
-export const SchemaBuilderCanvas = {
+export const VlsBuilderCanvas = {
   name: 'SchemaBuilderCanvas',
```
Comment (elena-vlasova) [inline packages/common/vue-layout-schema/lib/builder/components/canvas/vls-builder-canvas.js:20]: Name

File: packages/common/vue-layout-schema/lib/builder/components/canvas-tree/vls-builder-canvas-tree.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ import {
 import draggable from 'vuedraggable';
 import { SchemeView } from '../../mixins';
 
-export const SchemaBuilderCanvasTree = {
+export const VlsBuilderCanvasTree = {
   name: 'SchemaBuilderCanvasTree',
```
Comment (elena-vlasova) [inline packages/common/vue-layout-schema/lib/builder/components/canvas-tree/vls-builder-canvas-tree.js:15]: Name

File: packages/common/vue-layout-schema/lib/builder/components/blocks-list/vls-builder-blocks-list.js [hunk 5]
```diff
@@ -24,7 +24,7 @@ import { ATTR_TYPES } from '@deip/constants';
 import { convertBlockForSchema } from '../../utils/helpers';
 import { mutations } from '../../store';
 
-export const SchemaBuilderBlocksList = {
+export const VlsBuilderBlocksList = {
   name: 'SchemaBuilderBlocksList',
```
Comment (elena-vlasova) [inline packages/common/vue-layout-schema/lib/builder/components/blocks-list/vls-builder-blocks-list.js:28]: Name

**Review by elena-vlasova — CHANGES_REQUESTED** [review]

**Review by elena-vlasova — APPROVED** [review]

**Pull request merged**